.gitignore

```diff
@@ -61,3 +61,4 @@ Thumbs.db
 ehthumbs.db
 Desktop.ini
 $RECYCLE.BIN/
+!/perun-wui-cabinet/src/main/java/cz/metacentrum/perun/wui/cabinet/client/PerunCabinetView.ui.xml
```

Comment:
Shouldn't be here. This file is not excluded by default.

Reply:
I forgot to remove it. Now it should be OK.